src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(FIGURE_COUNT)) {
+            case 0:
+                return new Square(getRandomValue(), getRandomColor());
+            case 1:
+                return new Rectangle(getRandomValue(), getRandomValue(), getRandomColor());
+            case 2:
+                return new RightTriangle(getRandomValue(), getRandomValue(), getRandomColor());
+            case 3:
+                return new Circle(getRandomValue(), getRandomColor());
+            case 4:
+                return new IsoscelesTrapezoid(getRandomValue(), getRandomValue(), getRandomValue(),
+                        getRandomColor());
+            default:
+                return null;
+        }
+    }
+
+    private double getRandomValue() {
+        return random.nextDouble() * 10 + 1;
```

Comment:
Magic numbers 10 and 1, save it to constant! MIN_VALUE, MAX_VALUE

Reply:
not fixed

Reply:
not fixed